docs/enhancements/installercache.md

```diff
@@ -0,0 +1,51 @@
+# Installer Cache for Openshift Releases
```

Comment:
If you're going to put something in enhancements I think it should really follow the format and likely have a separate PR.

Alternatively, if you just want to write a dev doc (which I think is what you're doing) we have a separate directory for those (docs/dev)

Reply:
I'll put something in dev docs instead of this then.